### DISCUSSION TITLE
Monet on Android 12?

### DISCUSSION THREAD

**Discussion by Dubzer** [bromite/bromite#1585] [Q&A] [answered]
Hi. 
How can I enable dynamic colors based on wallpaper (monet)? I tried to use this flag: `chrome://flags/#dynamic-color-android`, but the browser is still uses the default colors.

Chrome by Google is using monet's color palette.

Bromite version: 95.0.4638.78

**Comment by Dubzer** [comment, marked as answer]
Fixed in the new version

**Reply by rodrigoswz** [reply]
I can confirm, Monet is finally on Bromite!

With Chromium v100 this should be even better, expanding to the menu and settings.

**Reply by srappan** [reply]
I can't seem to enable it. How did you get it working?
